miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppSecureStorageDispatcher.kt

```diff
@@ -2,38 +2,34 @@ package com.rakuten.tech.mobile.miniapp.js
 
 import android.app.Activity
 import androidx.annotation.VisibleForTesting
-import androidx.lifecycle.Observer
 import com.google.gson.Gson
 import com.rakuten.tech.mobile.miniapp.errors.MiniAppSecureStorageError
 import com.rakuten.tech.mobile.miniapp.storage.MiniAppSecureStorage
-import com.rakuten.tech.mobile.miniapp.storage.StorageState
+
+internal const val DB_NAME_PREFIX = "rakuten-"
```

Comment:
Can we rename it to `rakuten-miniapp-` ?

Reply:
Renamed it to `rmapp` since don't want to disclose `Miniapp` or something.